README.md

```diff
@@ -32,3 +32,10 @@ python dfusion.py
 # Verbose mode
 ./dfusion trades -v
 ```
+
+## Development
+
+```bash
+# Organize imports
+isort -rc */**.py
+```
```

Comment:
You can also add this to the editor on save actions, like what prettier does for js code formatting.

Maybe you'll need `isort` in your path and this setting turned on: https://github.com/microsoft/vscode-python/pull/1926#issuecomment-477340967

Reply:
https://github.com/gnosis/dex-cli/issues/21